src/gui/wizard/owncloudwizard.cpp

```diff
@@ -246,6 +246,8 @@ void OwncloudWizard::askExperimentalVirtualFilesFeature(const std::function<void
                "When a file is opened its contents will be downloaded automatically. "
                "Alternatively files can be downloaded manually by using their context menu."
                "\n\n"
+               "Switching to this mode will abort any currently running synchronization."
```

Comment:
(optional) Maybe we could be smart and show this message only if the sync is running